theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -217,17 +230,16 @@ def setup_sources_core(project, os, version, repoowner, pull_request = false) {
 
     dir(build_dir) {
         if (version == 'nightly') {
+            def build_vars = read_build_vars("../${project}")
             def url_base = 'https://ci.theforeman.org'
-            def job_name = nightly_jenkins_job
-            def job_id = nightly_jenkins_job_id
-            def job_url = "${url_base}/job/${job_name}/${job_id}"
+            def job_name = "${project}-develop-source-release"
+            def job_id = 'lastSuccessfulBuild'
+            def job_url = "${url_base}/job/${build_vars['source_location']}/${job_id}"
```

Comment:
And because I forgot to mention this in the first round: should this be `nightly_source_location`, or even `nightly_source_job` just to explicitly point out it's used for nightly only?

Reply:
`source_location` is what we use with RPM variables so I wanted to keep consistency

Reply:
ok